### PR TITLE
Update skill model / admin panel

### LINKED ISSUE #1076: Admin - Changes to skill model and admin panel
# Description
As shown below.

- [ ] This task requires a unit test.

# Images
[Skill Categorization tables.pdf](https://github.com/GCTC-NTGC/TalentCloud/files/3374258/Skill.Categorization.tables.pdf)

### PR DESCRIPTION
**Pull request by gggrant**
Resolves #1076.

### Notes:
- Migrations for new skill columns, classifications table and pivot table
- Update/add models
- Add new skill controls to admin panel

### CODE REVIEW

File: tests/Unit/Lang/LangFilesTest.php [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 class LangFilesTest extends BaseTranslationTest
 {
     public function testAllLangFilesWellFormatted()
     {
-        foreach ($this->locales as $locale) {
-            foreach ($this->getAllLangFilenames() as $langFile) {
-                $this->assertIsArray(Lang::get($langFile));
+        foreach ($this->getAllLangFilenames() as $langFile) {
+            if (!empty($langFile)) {
+                $this->assertInternalType('array', Lang::get($langFile));
```
Comment (Khristophor) [inline tests/Unit/Lang/LangFilesTest.php]: This line is throwing a warning about being deprecated in an upcoming PHPUnit 9 release, which is why I had changed it to `assertIsArray()`

Comment (gggrant) [reply tests/Unit/Lang/LangFilesTest.php]: Didn't mean to overwrite that.

File: database/migrations/2019_07_12_173700_add_classifications_table.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    public function up()
+    {
+        Schema::create('classifications', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('key');
+            $table->timestamps();
+        });
+
+        DB::table('classifications')->insert([
+            ['key' => 'BI'],
```
[the remark this answers is not in the document]
Comment (tristan-orourke) [reply database/migrations/2019_07_12_173700_add_classifications_table.php]: I added it in

**Comment by tristan-orourke** [comment]
> The CSS on the select feild isn't working right. I got this error, could you look into it?
> `Refused to load the stylesheet 'https://cdnjs.cloudflare.com/ajax/libs/select2-bootstrap-theme/0.1.0-beta.10/select2-bootstrap.min.css' because it violates the following Content Security Policy directive: "style-src 'self' 'unsafe-inline' https://fonts.googleapis.com https://code.ionicframework.com https://cdn.datatables.net https://stackpath.bootstrapcdn.com". Note that 'style-src-elem' was not explicitly set, so 'style-src' is used as a fallback.`

I just needed to restart Docker.

**Pull request merged**